P5/Source/Guidelines/en/FS-FeatureStructures.xml

```diff
@@ -269,17 +269,29 @@ combinations of feature names and values are used is to provide a
 <p>Whether at the level of feature-system declarations, feature- and
 feature-value libraries, or individual features, it is possible to
 align both feature names and their values with standardized external
-data category repositories such as ISOcat. <note place="bottom">See
+data category repositories. <note place="bottom">See
 section <ptr target="#DIMVLV"/> for more discussion of the need and
-rationale for ISOcat references.</note> In the following example, both
+rationale for aligning the content of grammatical (and other) descriptions with 
+standardized external taxonomies.</note> In the following example, both
 the feature <val>part_of_speech</val> and its value
-<val>#commonNoun</val> are aligned with the respective definitions
-provided by <ref target="#ISO-12620">ISO DCR (Data Category
-Registry)</ref>, as implemented by ISOcat.
-<egXML xmlns="http://www.tei-c.org/ns/Examples" valid="feasible" source="#UND">
-<fs xmlns:dcr="http://www.isocat.org/ns/dcr">
-<!--...-->
-<f name="part_of_speech" dcr:datcat="http://www.isocat.org/datcat/DC-1345" fVal="#commonNoun" dcr:valueDatcat="http://www.isocat.org/datcat/DC-1256"/>
+<val>NN</val> (standing for "common noun") are aligned with the respective definitions
+provided by the <ref target="https://www.clarin.eu/content/clarin-concept-registry">CLARIN Concept Registry (CCR)</ref>.
+<egXML xmlns="http://www.tei-c.org/ns/Examples">
+<fs>
+<f name="part_of_speech" datcat="http://hdl.handle.net/11459/CCR_C-396_5a972b93-2294-ab5c-a541-7c344c5f26c3">
+<symbol valueDatcat="http://hdl.handle.net/11459/CCR_C-1256_7ec6083c-23d4-224d-6f94-eecbe6861545" value="NN"/>
+</f>
+<!-- ... -->
+</fs>
+</egXML></p>
+<p>Since the above representation takes up a lot of space and quickly becomes redundant and
+error-prone, it is possible to delegate the task of aligning with external repositories to
+elements such as <gi>fLib</gi>, <gi>fvLib</gi>, <gi>fDecl</gi>, or <gi>fsDecl</gi> to reduce the feature
+representation at hand and to increase its readability at the same time, as shown below.
+ <egXML xmlns="http://www.tei-c.org/ns/Examples">
+<fs>
+<!-- ... -->
+<f name="POS" fVal="#commonNoun"/>
```

Comment:
NN instead of #commonNoun?

Reply:
Nope, that's the trick. It's a matter for a separate section (not necessarily of the Guidelines): you can't use "true" tagset labels in the fVal references, because some of them are not well-formed from the point of view of datatype constraints on URIs. So with fVal you are either lucky or you need to work through a redirection. The latter is a safer way. 
I'll demonstrate that separately, I have a little project on a side that studies that.

Reply:
ok, but where does the "#commonNoun" refer to? Do we need an explanatory sentence?

Reply:
+1 -- I'm now in the rendered version where it is easier to see which explanation belongs to which example.

Reply:
I'm not sure what "+1" means in this context :-)

Reply:
I will tentatively resolve this, to see how that affects the entire process (it's the last unresolved issue)